vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -77,6 +77,12 @@ See the [Build and Publish a Simple Bundle](../../tutorials/create/pb/publish-si
    - Component type and stack determines the build process, e.g. a React MFE executes an npm build. It can be customized in the command section of the `entando.json`.
    - All the components in the bundle are built in parallel.
    - A log file for each component is generated inside the .entando/logs directory of the project.
+- `--fail-fast`:
+   -  Fail the command as soon as any sub-task fails
+- `--max-parallel=<value>`:
+   -  Maximum number of sub-tasks running at the same time. The default value is 3. Setting `max-parallel=1` results in a sequential process.
```

Comment:
subtasks

(minor preference to use "simultaneously." nitpicky. can please ignore without prejudice.)